arviz/plots/compareplot.py

```diff
@@ -53,59 +61,70 @@ def plot_compare(comp_df, insample_dev=True, plot_standard_error=True, plot_ic_d
     yticks_pos, step = np.linspace(0, -1, (comp_df.shape[0] * 2) - 1, retstep=True)
     yticks_pos[1::2] = yticks_pos[1::2] + step / 2
 
-    yticks_labels = [''] * len(yticks_pos)
+    yticks_labels = [""] * len(yticks_pos)
 
-    information_criterion = 'waic'
+    information_criterion = "waic"
     if information_criterion not in comp_df.columns:
-        information_criterion = 'loo'
+        information_criterion = "loo"
 
     if plot_ic_diff:
         yticks_labels[0] = comp_df.index[0]
         yticks_labels[2::2] = comp_df.index[1:]
         ax.set_yticks(yticks_pos)
-        ax.errorbar(x=comp_df[information_criterion].iloc[1:],
-                    y=yticks_pos[1::2],
-                    xerr=comp_df.dse[1:],
-                    color=plot_kwargs.get('color_dse', 'grey'),
-                    fmt=plot_kwargs.get('marker_dse', '^'),
-                    mew=linewidth,
-                    elinewidth=linewidth)
+        ax.errorbar(
+            x=comp_df[information_criterion].iloc[1:],
+            y=yticks_pos[1::2],
+            xerr=comp_df.dse[1:],
+            color=plot_kwargs.get("color_dse", "grey"),
+            fmt=plot_kwargs.get("marker_dse", "^"),
+            mew=linewidth,
+            elinewidth=linewidth,
+        )
```

Comment:
@aloctavodia You and black are not going to be friends on the placement of parentheses either :(

Reply:
If black is automatic, maybe we shouldn't focus on the formatting side and consider more about the functionality. 🥇

Reply:
But we are humans! That is like saying we should focus on the nutritional value of food and not its presentation :-)